packages/@tinacms/cli/src/cmds/audit/audit.ts

```diff
@@ -11,232 +11,70 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { resolve } from '@tinacms/graphql'
-import type { Database, TinaCloudCollection } from '@tinacms/graphql'
-import p from 'path'
-import { logger } from '../../logger'
-import { assertShape } from '@tinacms/graphql'
-import chalk from 'chalk'
+import { AuditIssue } from './issue'
+import { validateRichText } from './validations/validateRichText'
+import { validateMutations } from './validations/validateMutations'
+import { AuditArgs, Resolver } from './auditArgs'
 
-type AuditArgs = {
-  collection: TinaCloudCollection
-  database: Database
-  rootPath: string
-  useDefaultValues: boolean
-}
-export const auditCollection = async (args: AuditArgs) => {
-  let warning = false
-  const { collection, database, rootPath } = args
-  logger.info(`Checking collection ${collection.name}`)
-  const query = `query {
-        collection(collection: "${collection.name}") {
-          format
-          documents {
-            edges {
-              node {
-                ...on Document {
-                  _sys {
-                    extension
-                    path
-                  }
+const interateCollectionDocuments = async (
```

Comment:
We were only auditing the first page of documents (10), so I had to introduce some logic to iterate through each page